client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEditForm.tsx

```diff
@@ -138,11 +126,11 @@ export const ResponseLineEditForm = ({
               <NumericTextInput
                 disabled={disabled}
                 autoFocus
-                value={numberOfPacksFromQuantity(supplyQuantity)}
+                value={supplyQuantity}
                 width={150}
                 onChange={q =>
                   update({
-                    supplyQuantity: q && numberOfPacksToTotalQuantity(q),
```

Comment:
I'm not entirely sure what the old code was doing here? wouldn't && make it a boolean?

Reply:
More JS joy... `&&` is more of a control flow thing - if the first value (`q`) isn't false/falsy, carry on and return the second value, otherwise return false 🫠